src/parser/wast-parser.cpp

```diff
@@ -391,24 +389,30 @@ Result<WASTCommand> command(Lexer& in) {
   return *mod;
 }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
Hmm, this keeps coming up... :smile: Oh well, this seems like the best solution, we don't want to disable this globally.